api/environments/models.py

```diff
@@ -405,6 +405,7 @@ def _get_environment_document_from_db(
                         "feature",
                         "feature_state_value",
                         "identity",
+                        "environment_feature_version",
```

Comment:
"We can include or exclude this based on whether `environment.use_v2_feature_versioning` is enabled, but I don't really see any downside to including it for environments where` use_v2_feature_versioning` is not enabled."

Reply:
agreed.